app/src/main/res/layout-sw600dp/vocable_confirmation_dialog.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It looks like this file is identical to `layout/vocable_confirmation_dialog`. If that's the case, you only need to keep the one in the base `layout` directory and it will get used by all form factors.